pkg/kev/kev_reconcile_test.go

```diff
@@ -273,10 +292,17 @@ var _ = Describe("Reconcile", func() {
 				})
 
 				It("should configure the added service labels from healthcheck config", func() {
```

Comment:
Title of this `It` should change as we don't deal with labels anymore.

Reply:
same here

Reply:
Similar comment to the one above here. Tiny text change so would address on the spot.